server/src/requirements.txt

```diff
@@ -1,9 +1,8 @@
-fastapi==0.61.1
-uvicorn==0.11.1
-python-multipart==0.0.5
-pyrovision == 0.1.2
-python-dotenv == 0.17.0
+fastapi>=0.61.1
+uvicorn>=0.11.1
+python-multipart>=0.0.5
+pyrovision
+python-dotenv >= 0.17.0
 pyroclient@git+https://github.com/pyronear/pyro-api.git#egg=pyroclient&subdirectory=client
-python-jose==3.2.0
-passlib[bcrypt]==1.7.4
-huawei-lte-api == 1.4.18
+python-jose>=3.2.0
+passlib[bcrypt]>=1.7.4
```

Comment:
That is a bit risky: I understand for provision that it was the only solution, but for the rest, since this is production environment, and we've encountered version issues in the past, I'd suggest sticking to full version pinning :sweat_smile:

Reply:
ok I get your point

Reply:
Done !